utils/rpms/daos.spec

```
@@ -317,6 +321,9 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 %{_libdir}/*.a
 
 %changelog
+* Mon Jul 20 2020 Maureen Jean <maureen.jean@intel.com> - 1.0.1-2
+- Include correct SLES fuse lib package
```

Technically, probably should be:

```suggestion
- Include correct Suse fuse lib package
```

since this applies to more than SLES.

But certainly no need to push a new patch just for that.

---

I will fix this